include/deal.II/grid/grid_tools.h

```diff
@@ -1127,9 +1127,11 @@ namespace GridTools
    * first_vector_components can be specified that describes how vector valued
    * DoFs of the first face should be modified prior to constraining to the
    * DoFs of the second face. If @p first_vector_components is non empty the
-   * matrix is interpreted as a rotation matrix that is applied to all vector
-   * valued blocks listed in @p first_vector_components of the FESystem. For
-   * more details see make_periodicity_constraints() and the glossary
+   * matrix is interpreted as a rotation matrix that is applied to all
+   * vector valued blocks listed in @p first_vector_components of the
+   * FESystem. For more details see
```

Comment:
It could be an FESystem but doesn't have to be. It could also be some weird element that happens to have a lot of components.